custom/icds_reports/utils/__init__.py

```diff
@@ -233,11 +227,6 @@ def _custom_data(self, selected_location, domain):
             tags = ["location_type:{}".format(loc_type), "report_slug:{}".format(self.slug), "config:{}".format(config['id'])]
             if allow_conditional_agg:
                 tags.append("allow_conditional_agg:yes")
-            datadog_histogram(
-                "commcare.icds.block_reports.ucr_querytime",
-                timer.duration,
-                tags=tags
-            )
```

Comment:
not referenced in any DD dashboard. @sravfeyn are these still in use at all?

Reply:
Fine to remove